klibs.md

```diff
@@ -712,6 +733,7 @@ Certain caveats to be aware of:
 * When cloud_init cannot download one or more files, the kernel does not start the user program.
   The rationale for this is that we want all files to be ready and accessible when the program starts.
 * When used to populate the user environment, only string-valued attributes are converted to environment variables (**non-string-valued attributes are ignored**).
+* `"IMAGE_NAME"` environment variable, used by `syslog` klib, can't be set (has no effect) from `download_env`, since `syslog` is loaded much earlier then `cloud_init`.
```

Comment:
I would make this sentence more generic, since it can apply to other environment variables and other klibs. Something along the lines of:

> The cloud_init klib is not meant to be used to set environment variables whose value is used in other klibs or in the kernel code (e.g. the "IMAGE_NAME" environment variable used by the syslog klib), because the code that uses an environment variable can be executed before cloud_init sets a value for the variable.